contracts/ColonyTask.sol

```diff
@@ -228,29 +229,29 @@ contract ColonyTask is ColonyStorage, DSMath {
     bytes32 ratingSecret = generateSecret(_salt, _rating);
     require(ratingSecret == taskWorkRatings[_id].secret[_role]);
 
-    Role storage role = tasks[_id].roles[_role];
-    role.rated = true;
-    role.rating = _rating;
+    uint8 rating = convertRating(_rating);
+    require(rating != NONE, "Cannot rate None!");
```

Comment:
With the `convertRating` function, this require can never not be true. Arguably worth keeping in to remind us when we remove the old-style ratings entirely though (and thus the convert function).

Reply:
Mm that's true, I can remove it in the backwards-compatibility commit (it'll come back when we roll back).